<?xml version='1.0'?>
<launch>

    <node name="map_publisher" pkg="map_tile_loader" type="map_publisher" output="screen">
        <param name="map_path" type="string" value="/home/biomotion/nuscenes_maps" />
    </node>


</launch>
